presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloSplitManager.java

```diff
@@ -57,7 +57,7 @@ public AccumuloSplitManager(
     }
```

Comment:
I'd change the title of the commit message to "Pass split scheduling strategy to ConnectorSplitManager.getSplits"